<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BearingDesignUI</class>
 <widget class="QMainWindow" name="BearingDesignUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>743</width>
    <height>398</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>轴承设计UI</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <item>
     <widget class="QSplitter" name="splitter_2">
      <property name="orientation">
       <enum>Qt::Orientation::Horizontal</enum>
      </property>
      <widget class="QScrollArea" name="scrollArea">
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="scrollAreaWidgetContents">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>323</width>
          <height>342</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QGroupBox" name="groupBox_3">
           <property name="title">
            <string>设计参数(一)</string>
           </property>
           <layout class="QGridLayout" name="gridLayout" columnstretch="1,1,1,1">
            <item row="1" column="1">
             <widget class="QDoubleSpinBox" name="spinC">
              <property name="decimals">
               <number>1</number>
              </property>
              <property name="maximum">
               <double>200.000000000000000</double>
              </property>
              <property name="value">
               <double>61.799999999999997</double>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QPushButton" name="btnLife">
              <property name="text">
               <string>预期寿命(h)</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1" colspan="3">
             <widget class="QComboBox" name="comboBearing">
              <property name="currentIndex">
               <number>3</number>
              </property>
              <item>
               <property name="text">
                <string>调心球轴承 （1）</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>调心滚子轴承（2）</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>圆锥滚子轴承 （3）</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>深沟球轴承 （6）</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>角接触球轴承 （C）</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>角接触球轴承  (AC)</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>角接触球轴承  （B)</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>圆柱滚子轴承  (N)</string>
               </property>
              </item>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QDoubleSpinBox" name="spinLife">
              <property name="decimals">
               <number>0</number>
              </property>
              <property name="maximum">
               <double>3000000.000000000000000</double>
              </property>
              <property name="value">
               <double>5000.000000000000000</double>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>基本额定动载荷(KN)</string>
              </property>
             </widget>
            </item>
            <item row="1" column="2">
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>基本额定静载荷(KN)</string>
              </property>
             </widget>
            </item>
            <item row="1" column="3">
             <widget class="QDoubleSpinBox" name="spinCo">
              <property name="decimals">
               <number>1</number>
              </property>
              <property name="maximum">
               <double>200.000000000000000</double>
              </property>
              <property name="value">
               <double>38.000000000000000</double>
              </property>
             </widget>
            </item>
            <item row="2" column="3">
             <widget class="QDoubleSpinBox" name="spin_d">
              <property name="decimals">
               <number>0</number>
              </property>
              <property name="maximum">
               <double>150.000000000000000</double>
              </property>
              <property name="value">
               <double>50.000000000000000</double>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>轴承类型：</string>
              </property>
             </widget>
            </item>
            <item row="2" column="2">
             <widget class="QLabel" name="label_10">
              <property name="text">
               <string>轴承内径</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_5">
           <property name="font">
            <font>
             <pointsize>12</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>点击按钮即可进行查表操作，注意轴承类型</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_4">
           <property name="title">
            <string>设计参数(二)</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_2" columnstretch="1,1,1,1">
            <item row="2" column="2">
             <widget class="QPushButton" name="btnY">
              <property name="text">
               <string>轴向动载荷系数Y</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QDoubleSpinBox" name="spinX">
              <property name="decimals">
               <number>3</number>
              </property>
              <property name="value">
               <double>0.560000000000000</double>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QPushButton" name="btn_f">
              <property name="text">
               <string>载荷系数fd</string>
              </property>
             </widget>
            </item>
            <item row="3" column="2" colspan="2">
             <widget class="QPushButton" name="btnFd">
              <property name="text">
               <string>派生轴向力公式</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QPushButton" name="btnX">
              <property name="text">
               <string>径向动载荷系数X</string>
              </property>
             </widget>
            </item>
            <item row="2" column="3">
             <widget class="QDoubleSpinBox" name="spinY">
              <property name="decimals">
               <number>3</number>
              </property>
              <property name="value">
               <double>1.597000000000000</double>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QDoubleSpinBox" name="spin_f">
              <property name="decimals">
               <number>1</number>
              </property>
              <property name="maximum">
               <double>99.000000000000000</double>
              </property>
              <property name="value">
               <double>1.200000000000000</double>
              </property>
             </widget>
            </item>
            <item row="0" column="3">
             <widget class="QDoubleSpinBox" name="spinFa">
              <property name="maximum">
               <double>120000.000000000000000</double>
              </property>
              <property name="value">
               <double>2700.000000000000000</double>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QDoubleSpinBox" name="spinFr">
              <property name="maximum">
               <double>120000.000000000000000</double>
              </property>
              <property name="value">
               <double>5500.000000000000000</double>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QPushButton" name="btn_e">
              <property name="text">
               <string>判断系数e</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QDoubleSpinBox" name="spin_e">
              <property name="value">
               <double>0.490000000000000</double>
              </property>
             </widget>
            </item>
            <item row="1" column="3">
             <widget class="QDoubleSpinBox" name="spin_n">
              <property name="maximum">
               <double>5000.000000000000000</double>
              </property>
              <property name="value">
               <double>1250.000000000000000</double>
              </property>
             </widget>
            </item>
            <item row="0" column="2">
             <widget class="QLabel" name="label_7">
              <property name="text">
               <string>轴向载荷Fa(N)</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="label_8">
              <property name="text">
               <string>径向载荷Fr(N)</string>
              </property>
             </widget>
            </item>
            <item row="1" column="2">
             <widget class="QLabel" name="label_9">
              <property name="text">
               <string>轴承转速(r/min)</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label">
           <property name="font">
            <font>
             <pointsize>12</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>基本额定载荷可查询该网站：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_6">
           <property name="font">
            <font>
             <pointsize>12</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>https://www.skf.com/cn/products/rolling-bearings</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QSplitter" name="splitter">
       <property name="orientation">
        <enum>Qt::Orientation::Vertical</enum>
       </property>
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>设计图表</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QScrollArea" name="scrollArea_2">
           <property name="widgetResizable">
            <bool>true</bool>
           </property>
           <widget class="QWidget" name="scrollAreaWidgetContents_2">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>829</width>
              <height>322</height>
             </rect>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <item>
              <widget class="QLabel" name="labPic">
               <property name="text">
                <string/>
               </property>
               <property name="pixmap">
                <pixmap resource="res.qrc">:/images/images/Bearing_Lh.png</pixmap>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="title">
         <string>输出文本框</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QPlainTextEdit" name="textEdit"/>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonStyle::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actStart"/>
   <addaction name="separator"/>
   <addaction name="actShow"/>
   <addaction name="actClear"/>
   <addaction name="separator"/>
   <addaction name="actExit"/>
  </widget>
  <action name="actStart">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/images/wizard24.png</normaloff>:/images/images/wizard24.png</iconset>
   </property>
   <property name="text">
    <string>开始设计</string>
   </property>
   <property name="toolTip">
    <string>开始设计</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actShow">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/images/704.bmp</normaloff>:/images/images/704.bmp</iconset>
   </property>
   <property name="text">
    <string>显示设计</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actClear">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/images/103.bmp</normaloff>:/images/images/103.bmp</iconset>
   </property>
   <property name="text">
    <string>清除输出</string>
   </property>
   <property name="toolTip">
    <string>清除输出</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actExit">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/images/exit.bmp</normaloff>:/images/images/exit.bmp</iconset>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
   <property name="toolTip">
    <string>退出程序</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actExit</sender>
   <signal>triggered()</signal>
   <receiver>BearingDesignUI</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>371</x>
     <y>198</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actClear</sender>
   <signal>triggered()</signal>
   <receiver>textEdit</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>535</x>
     <y>309</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
